.github/scripts/check_readme.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+# This script will verify that the README.md of all task and 
+# pipeline directories provided matches the output of hack/readme_generator.sh
+# to ensure that README files are up to date.
+#
+# Task and pipeline directories are provided
+# either via README_ITEMS env var, or as arguments
+# when running the script.
+
+# yield empty strings for unmatched patterns
+shopt -s nullglob
+
+CLI_README_ITEMS=""
+
+show_help() {
+  echo "Usage: $0 [item1] [item2] [...]"
+  echo
+  echo Flags:
+  echo "  --help: Show this help message"
+  echo
+  echo "Items are task or pipeline directories. They can be supplied"
+  echo "either as arguments or via the README_ITEMS environment variable."
+  echo
+  echo "Examples:"
+  echo "  $0 tasks/managed/apply-mapping"
+  echo
+  echo "  or"
+  echo
+  echo '  export README_ITEMS="mydir/tasks/apply-mapping some/other/dir"'
+  echo "  $0"
+  exit 1
+}
+
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    --help)
+      show_help
+      ;;
+    --*)
+      show_help
+      ;;
+    *)
+      CLI_README_ITEMS+="$1 "
```

Comment:
You could make CLI_README_ITEMS an array, to avoid potential issues like if a directory contains a space or something.
```suggestion
      CLI_README_ITEMS+=("$1")
```

Reply:
fixed